src/coreclr/vm/methodtable.cpp

```diff
@@ -3443,6 +3443,15 @@ int MethodTable::GetLoongArch64PassStructInRegisterFlags(CORINFO_CLASS_HANDLE cl
                         size = STRUCT_FIRST_FIELD_SIZE_IS8;
                     }
                 }
+                else if (fieldType == ELEMENT_TYPE_CLASS)
+                {
+                    // Here is just skip the array as its elements' count greater than 1.
+                    // TODO-LoongArch64: liking `struct {int array[1]; float field_2;}` which the array field is only one element,
```

Comment:
Do you expect that this TODO can be ever fixed? I do not think that it is really fixable. I think it would be better to delete the TODO comment.

Reply:
OK, Thanks